src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -242,6 +244,7 @@ def make_parameter_file(
         'Azimuth bandpass filter: no\n',
         f'DEM source: {dem_name}\n',
         f'DEM resolution (m): {dem_resolution}\n',
+        f'Pixel Size (m): {pixel_size}\n',
```

Comment:
I vote against adding the pixel size to the parameter .txt file unless someone has a compelling use case for it. Pixel size is a self-describing property of the output geotiffs, so I don't think the extra code to include it in the txt file is justified. Thoughts?

Reply:
will delete the pixel size in the txt file